src/tip.js

```diff
@@ -139,7 +139,7 @@ function tokenizeTip(eat, value, silent) {
       },
       {
         type: "text",
-        value: title
+        value: title || DEFAULT_TITLE[tipType]
```

Comment:
I think in the `guide` case, this value is `undefined`. Is that what we want?

Tested on a JS console:
```
> const DEFAULT_TITLE = { type: "Teaching Tip", discussion: "Discussion Goal" }
undefined
> const title = "";
undefined
> title || DEFAULT_TITLE['guide']
undefined
```

Reply:
Hmmm, looking at the original code I actually think what we want here is in fact the string "None" :vomiting_face:

Reply:
Oh, actually we want to avoid rendering the paragraph entirely unless there is a defined title. I added a test to CB itself to capture that behavior (https://github.com/code-dot-org/curriculumbuilder/pull/295); I'll add another one here